release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1309,6 +1313,63 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously, the technology preview oc-mirror plug-in v2 feature failed to mirror operator catalogs from disk-to-mirror in fully disconnected environments. This issue also affected catalogs that specified a `targetCatalog` in the `ImageSetConfiguration` file. With this update, oc-mirror v2 can now successfully mirror catalogs in fully disconnected environments, and the `targetCatalog` functionality works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34521[*OCPBUGS-34521*])
+
+* Previously, the `oc adm must-gather` command took a long time to gather CPU related performance data in large clusters. With this release, the data is gathered in parallel instead of sequentially, which shortens the data collection time. (link:https://issues.redhat.com/browse/OCPBUGS-34360[*OCPBUGS-34360*])
+
+* Previously, in the technology preview oc-mirror plug-in v2 feature, errors encountered on registries, such as timeouts, expired authentication tokens, HTTP 500 errors, and so on were not retried automatically. With this update, retries for these errors have been implemented. With this update, retries for these errors are implemented, and users can configure retry behavior with the following flags:
++
+* `--retry-times`: Specifies the number of retry attempts. Default is 2.
+* `--retry-delay`: Sets the delay between retries. Default is 1 second.
+* `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
+* `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-34021[*OCPBUGS-34021*])
+
+* Previously, the technology preview oc-mirror plug-in v2 feature did not generate `ImageDigestMirrorSet` (IDM) or `ImageTagMirrorSet` (ITM) files when mirroring failed due to network issues or invalid operator catalogs. With this update, oc-mirror continues mirroring other images when operator or additional images fail, and only stops when release images fail. Cluster resources are generated based on successfully mirrored images, and all errors are collected in a log file for review.
+(link:https://issues.redhat.com/browse/OCPBUGS-34020[*OCPBUGS-34020*])
+
+* Previously, in the technology preview oc-mirror plug-in v2 feature, there was no validation for the `-v2` vs `--v2` flags. As a result, users who mistakenly used `-v2`, that sets the log level to 2 instead of `--v2`, that switches to oc-mirror v2, encountered unclear error messages. With this update, flag validation is provided. If `-v2` is used while the `ImageSetConfig` is using the `v2alpha1` API and `--v2` is not specified, an error message is displayed. The following message is now enabled that provides a clear guidance to the user:
```

Comment:
```suggestion
* Previously, there was no validation for the `-v2` vs `--v2` flags for the `oc-mirror` command. As a result, users who mistakenly used `-v2`, that sets the log level to 2 instead of `--v2`, which switches to `oc-mirror` v2 plugin, causing unclear error messages. With this update, flag validation is provided. If `-v2` is used while the `ImageSetConfig` is using the `v2alpha1` API and `--v2` is not specified, an error message is displayed. The following message is now enabled that provides a clear guidance to the user:
```